src/test/resources/liquibase/ext/changelog.empty.yaml

```diff
@@ -0,0 +1,2 @@
+databaseChangeLog:
+  - isEmpty: 1 # needed so that liquibase sees the databaseChangeLog as a list
```

Comment:
is there a better way of creating an empty changelog?

Reply:
Just `databaseChangeLog:` should be enough.. I think we fixed that already.

Reply:
that now works, yes, thanks!